spring-cloud-gcp-docs/src/main/asciidoc/config.adoc

```diff
@@ -95,34 +95,29 @@ above `SampleConfig` bean.
 === Refreshing the configuration at runtime
 
 http://cloud.spring.io/spring-cloud-static/docs/1.0.x/spring-cloud.html#_endpoints[Spring
-Boot Actuator] enables a `/refresh` endpoint in your application that can be used to refresh the
-values of all the Spring Cloud Config-managed variables.
+Cloud] provides support to have configuration parameters be
+reloadable with the POST request to `/actuator/refresh` endpoint.
 
-To achieve that, add the Spring Boot Actuator dependency.
-
-Maven coordinates:
-
-----
-<dependency>
+1.  Add the following dependency to your `pom.xml`:
++
+....
+  <dependency>
     <groupId>org.springframework.boot</groupId>
     <artifactId>spring-boot-starter-actuator</artifactId>
-</dependency>
-----
-
-Gradle coordinates:
-
-[source,subs="normal"]
-----
-dependencies {
-    compile group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'
-}
-----
-
-Add a `@RefreshScope` annotation to your class(es) containing remote configuration properties.
-Then, if you change the value of the `myapp.queue-size` variable in the `myapp_prod`
-configuration and hit the `/refresh` endpoint of your application, you can verify that the
-value of the `queueSize` field has been updated.
-
-NOTE: If you're developing locally or just not using authentication in your application, you should
-add `management.security.enabled=false` to your `application.properties` file to allow unrestricted
-access to the `/refresh` endpoint.
+  </dependency>
+....
+2.  Add `@RefreshScope` to your Spring configuration class to have parameters be reloadable at runtime.
+3.  If your application doesn't require authentication, add `management.endpoints.web.expose=*`
```

Comment:
We probably don't want to recommend `management.endpoints.web.expose=*` as it's too permissive.
See: https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-endpoints.html#production-ready-endpoints-enabling-endpoints

Also, according to current documentation and our sample app, that property is incorrect.
The correct property is `management.endpoints.web.exposure.include`, and I think it's `management.endpoints.web.exposure.include=env` is what we need. Please test with `spring-cloud-gcp-config-sample`.